docs/releases/2.7.0.rst

```diff
@@ -25,6 +25,8 @@ Major Changes
 - Backgrounding statistics calculations - so Pootle and its users are never
   bogged down or delayed by real time stats calculations.  Instead these are
   backgrounded and updated when available.
+- A number of features have been dropped and will be recovered in future
```

Comment:
Might be worth mentioning somewhere what's not available anymore, as these can be blockers for people to upgrade.